testsuite/features/step_definitions/navigation_steps.rb

```diff
@@ -5,6 +5,13 @@
 # Texts and links
 #
 
+def wait_for_spa_transition(text)
```

Comment:
I don't like the name of this method here, we might want to use something more understandable from the point of view of a user. Something like `wait_page_loaded` could make sense?

Reply:
For me both look ok, I can change it to wait_page_loaded if you prefer